src/Germany/Mews.Fiscalizations.Germany/Client.cs

```diff
@@ -26,7 +26,7 @@ internal Client()
             where TResult : class
         {
             var httpResponse = await SendRequestAsync(method, endpoint, request, token).ConfigureAwait(continueOnCapturedContext: false);
-            return await DeserializeAsync(httpResponse, successFunc);
+            return await DeserializeAsync(httpResponse, successFunc).ConfigureAwait(continueOnCapturedContext: false);
```

Comment:
Or we could just do this (everywhere). I know we didn't yet tackle this in a systematic way, but can we at least agree on not awaiting something for no reason at all? @starychfojtu @JakubLinhart 
```suggestion
            return DeserializeAsync(httpResponse, successFunc);
```

Reply:
The method is declared async, so I believe it's not a good practice to not use await in some places (for example in this case to deserialize the response).
and this method is declared async because the code is awaiting for a response a few lines above. (SendRequestAsync)

Reply:
My bad. In this case the compiler doesn't let you not await the result as we already await above. The map would be a good option if available to us as we wouldn't need to await.